src/libServer/LookupServer.cpp

```diff
@@ -1369,13 +1371,13 @@ Json::Value LookupServer::GetShardingStructure() {
 
     unsigned int num_shards = shards.size();
 
-    if (num_shards == 0) {
+    /* if (num_shards == 0) {
```

Comment:
I think can keep this check still?

Reply:
Not really sure.. Do we want to alert user when no shard exist situation could be valid. Since dscommitte always exist. If it’s used only by explorer, we can keep this check.

Reply:
If you see the API, it will just return an empty object if there are no shards.

Reply:
@KaustubhShamshery , `LookupServer::GetBlockchainInfo()` currently fails if we raise exception for `No Shard Yet` and explorer dont update the info. So, we need to fall back where we don't raise exception because its completely valid to have no shards. Other option needs evaluation on api or explorer itself.